internal/object.go

```diff
@@ -165,3 +172,51 @@ func setObject(ctx context.Context, doc *enigma.Doc, info *enigma.NxInfo, props
 	}
 	return nil
 }
+
+func Publish(ctx context.Context, doc *enigma.Doc, objectID string) error {
+	object, err := doc.GetObject(ctx, objectID)
+	if err != nil {
+		return err
+	}
+	if object.Handle != 0 {
+		var layout Layout
+		var raw json.RawMessage
+		raw, err = object.GetLayoutRaw(ctx)
+		err = json.Unmarshal(raw, &layout)
+		if layout.Meta.Published {
+			log.Infoln(objectID + " is published.")
+			return fmt.Errorf("Cannot publish a published object: " + objectID)
```

Comment:
style: Error messages in go should begin with lowercase letters.

Reply:
okey dokey.